deployment/codefresh-dev.yaml

```diff
@@ -247,4 +247,4 @@ steps:
         candidate: '${{REGISTRY}}/salk/workspaces:${{CF_BUILD_ID}}'
```

Comment:
please be carefull about adding this file to the commit, most times you don't want it to be in the commit